mlflow/server/auth/__init__.py

```diff
@@ -552,12 +553,18 @@ def _after_request(resp: Response):
 
 def create_admin_user(username, password):
     if not store.has_user(username):
-        store.create_user(username, password, is_admin=True)
-        _logger.info(
-            f"Created admin user '{username}'. "
-            "It is recommended that you set a new password as soon as possible "
-            f"on {UPDATE_USER_PASSWORD}."
-        )
+        try:
+            store.create_user(username, password, is_admin=True)
+            _logger.info(
+                f"Created admin user '{username}'. "
+                "It is recommended that you set a new password as soon as possible "
+                f"on {UPDATE_USER_PASSWORD}."
+            )
+        except sqlalchemy.exc.IntegrityError:
```

Comment:
Are we sure if a `sqlalchemy.exc.IntegrityError` is thrown then the 'create admin user' at least succeeds once?

Reply:
Yes :)